### PR TITLE
Blockwriter: use new UUID

### PR DESCRIPTION
**Pull request by BramGruneir**
Fixes cockroachdb/cockroach#4420

### CODE REVIEW

File: block_writer/main.go [hunk 1]
```diff
@@ -67,7 +67,7 @@ func newBlockWriter(db *sql.DB) blockWriter {
 	source := rand.NewSource(int64(time.Now().UnixNano()))
 	return blockWriter{
 		db:   db,
-		id:   uuid.NewUUID4().String(),
+		id:   uuid.NewV4().String(),
```
Comment (tamird) [inline block_writer/main.go]: yikes. you should just use the upstream package directly here.

**Comment by tschottdorf** [comment]
not sure about the test failure, `./build/circle-test.sh` builds locally (with much the same output, except for not failing).

File: block_writer/main.go (hunk 1 above)
Comment (BramGruneir) [inline block_writer/main.go]: done

**Comment by tschottdorf** [comment]
LGTM

**Pull request merged**